main.go

```diff
@@ -35,10 +35,12 @@ var (
 )
 
 type Job struct {
-	JobName        string `yaml:"name"`
-	Image          string `yaml:"image"`
-	Namespace      string `yaml:"namespace"`
-	ServiceAccount string `yaml:"service_account,omitempty"`
+	JobName        string   `yaml:"name"`
+	Image          string   `yaml:"image"`
+	Namespace      string   `yaml:"namespace"`
+	ServiceAccount string   `yaml:"service_account,omitempty"`
+	Command        []string `yaml:"command"`
```

Comment:
We could use `,omitempty` here since these are not required fields, for if we decide to serialise the YAML back to disk at any point. Namespace should also be optional.

Like: https://github.com/openfaas/faas-provider/blob/master/types/function_deployment.go

Reply:
Added. I guess in this case we don't need to do any additional checks, right?